tests/test_losses.py

```diff
@@ -0,0 +1,19 @@
+import torch
+import pytest
+import numpy as np
+
+from brats.losses import DiceLossOneClass
+
+BATCH_DIMS = (2, 5, 5, 5)
+
+
+class TestDiceLossOneClass:
```

Comment:
I think one more test with real values should be performed. Calculate something for small tensors and see if its correct. Cases for ideal and worst fit only might give you a false confidence of properly working function.

Reply:
Great!